src/index.ts

```diff
@@ -71,4 +71,15 @@ app.use('/graphql', graphqlExpress(req => ({
 
 app.use('/graphiql', graphiqlExpress({ endpointURL: '/graphql' }));
 
+app.use((err, req, res, next) => {
+  const { statusCode, message, stack } = err;
+  res.status(statusCode).json({
+    code: statusCode,
+    result: config.get('server.showErrorStack') ? `
+      message: ${message};
+      stack: ${stack}
+    ` : message
```

Comment:
Could this be a bit better as the return is a bit better to handle on the other side as you know you always have a message key in the result and with the error trace an array.

```suggestion
    result: config.get('server.showErrorStack') ?  {
      message,
      stack: stack.split(/\r?\n/).map(string => string.trim()).filter(string => string != "")
    }  : { message }
```

Reply:
I wanted to keep same structure for response, where `result` is a string https://github.com/DivanteLtd/vue-storefront-api/blob/master/src/lib/util.js#L73 . It's not declared here that it's a string, but this is how it is used in code. wdyt about this
```
const stackTrace = stack
  .split(/\r?\n/)
  .map(string => string.trim())
  .filter(string => string != "")

res.status(statusCode).json({
  code: statusCode,
  result: message,
  ...(config.get('server.showErrorStack') ? { stack: stackTrace } : {})
})
```

Reply:
That would work too.

But from the way it is implemented in the code you have linked, the result could be an string or object, even the jsDocs suggest it `@param {json} [result='OK']    Text message or result information object` or are the jsdocs wrong or misleading?

Reply:
Oww yeah, you are right. I don't like it :D In reality it has `any` type, because this object is not specified. I will stay with `{code: number, result: string, stack?: string[]}`ノ( º _ ºノ)